packages/cli/build/main.ts

```diff
@@ -35,9 +38,12 @@ async function main(mode) {
     args.filter((x) => !x.startsWith('-')).forEach((target) => {
         command.push('--env', target)
         if (!knownTargets.includes(target)) {
-            throw new TypeError('Unknown target ' + target + '. Known targets: ' + knownTargets.join(','))
+            throw new TypeError(`Unknown target ${target}. Known targets: ${knownTargets}`)
```

Comment:
`knownTargets`, javascript array `toString` default behavior is use comma join string.

```
> foo = [1, 2, 3]
> `${foo}`
< "1,2,3" 
```